share/spice/statista/statista.js

```diff
@@ -0,0 +1,88 @@
+(function (env) {
+    "use strict";
+    
+    function getImage(item, size, blank) {  
+       if (item.teaserImageUrls[size]) {
+            var img = item.teaserImageUrls[size].src;
+            if (blank == 1) {
+                img = img + '?blank=blank';
+            }
+            return img;
+        } else if (size == 1) {
+            return 'http://statistacloudfront.s3.amazonaws.com/Statistic/table/table-355-1.png';
+        } else if (size == 2) {
+            return 'https://static1.statista.com/Statistic/table/table-100-1.png';
+        }
+    }
+    
+    function getTitle(title) {
+        return title.replace(/\ \|\ .+?$/, "");
+    }
+    
+    function formatDate(date) {
```

Comment:
We actually have Moment.js available to devs. If you wrap the `Spice.add()` in `DDG.require('moment.js', function() { ... })` inside the block, you'll have access to `moment` which will let you format dates very easily.

Here's an example: https://github.com/duckduckgo/zeroclickinfo-spice/blob/1f79de6380bd3c963c6c4e2e1fef86514b20fe31/share/spice/wikinews/wikinews.js#L9